h5netcdf/legacyapi.py

```diff
@@ -31,32 +33,34 @@ def __setattr__(self, name, value):
 
 
 class Variable(core.BaseVariable, HasAttributesMixin):
-    _cls_name = 'h5netcdf.legacyapi.Variable'
+    _cls_name = "h5netcdf.legacyapi.Variable"
 
     def chunking(self):
         chunks = self._h5ds.chunks
         if chunks is None:
-            return 'contiguous'
+            return "contiguous"
         else:
             return chunks
 
     def filters(self):
         complevel = self._h5ds.compression_opts
-        return {'complevel': 0 if complevel is None else complevel,
-                'fletcher32': self._h5ds.fletcher32,
-                'shuffle': self._h5ds.shuffle,
-                'zlib': self._h5ds.compression == 'gzip'}
+        return {
+            "complevel": 0 if complevel is None else complevel,
+            "fletcher32": self._h5ds.fletcher32,
+            "shuffle": self._h5ds.shuffle,
+            "zlib": self._h5ds.compression == "gzip",
+        }
 
     @property
     def dtype(self):
         dt = self._h5ds.dtype
-        if h5py.check_dtype(vlen=dt) is unicode:
-            return str
+        # if h5py.check_dtype(vlen=dt) is unicode:
+        #    return str
```

Comment:
I think we still want this for compatibility with netCDF4-python?

Reply:
Yes, forgot to fix this one...